experimental/addons/istio/examples/demo.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: website
+  namespace: default
+spec:
+  components:
+    - name: frontend
+      type: webservice
+      properties:
+        image: docker.io/kennethreitz/httpbin
+        ports:
+          - port: 80
+        cpu: "0.1"
+        env:
+          - name: FOO
+            value: bar
+      traits:
+          # use expose trait to create a svc
+        - type: expose
+          properties:
+            port:
+              - 80
+        - type: istio-gateway
+          properties:
+            hosts:
+              - awesomesite.com
+            # Default value is ingressgateway which is the default selector for your istio svc
```

Comment:
IMO example files should move from the addon dir to root dir. `/example/istio/....`

Reply:
commit:
44426672ccaf2098120d58c40c47ea42a344df71
moved to root directory